### PR TITLE
addressing Ben's final comments

### PR DESCRIPTION
**Pull request by seanturner**
Update reference.

### CODE REVIEW

File: draft-ietf-tls-certificate-compression.md [hunk 1]
```diff
@@ -141,10 +143,11 @@ compressed with the Zstandard compression algorithm as defined in [RFC8478].
 
 It is possible to define a certificate compression algorithm that uses a
 pre-shared dictionary to achieve higher compression ratio.  This document does
-not define any such algorithms.
+not define any such algorithms, but additional codepoints may be allocated for
+such use per the policy in section 7.3.
 
 If the received CompressedCertificate message cannot be decompressed, the
-connection MUST be torn down with the "bad_certificate" alert.
+connection MUST terminate the connection with the "bad_certificate" alert.
```
Comment (ghedo) [inline draft-ietf-tls-certificate-compression.md]:
```suggestion
connection MUST be terminated with the "bad_certificate" alert.
```

File: draft-ietf-tls-certificate-compression.md [hunk 2]
```diff
@@ -127,8 +128,9 @@ uncompressed_length
   enforce limits on the message size before performing decompression.
 
 compressed_certificate_message
-: The compressed body of the Certificate message, in the same format as it
-  would normally be expressed in. The compression algorithm defines how the
+: The result of applying the indicated compression algorithm to the encoded
+  Certificate message that would be sent if certificate compression was not
```
Comment (vasilvv) [inline draft-ietf-tls-certificate-compression.md]: would have been...?  (also, "were not in use"?)

File: draft-ietf-tls-certificate-compression.md (hunk 1 above)
Comment (vasilvv) [inline draft-ietf-tls-certificate-compression.md]: This currently says "connection MUST terminate the connection"

File: draft-ietf-tls-certificate-compression.md [hunk 3]
```diff
@@ -141,10 +143,11 @@ compressed with the Zstandard compression algorithm as defined in [RFC8478].
 
 It is possible to define a certificate compression algorithm that uses a
 pre-shared dictionary to achieve higher compression ratio.  This document does
-not define any such algorithms.
+not define any such algorithms, but additional codepoints may be allocated for
+such use per the policy in section 7.3.
```
Comment (vasilvv) [inline draft-ietf-tls-certificate-compression.md]: Could you replace this with a `{{section_name}}` reference?

File: draft-ietf-tls-certificate-compression.md [hunk 4]
```diff
@@ -157,23 +160,23 @@ encoded without being compressed.  This way, the parsing and the verification
 have the same security properties as they would have in TLS normally.
 
 In order for certificate compression to function correctly, the underlying
-compression algorithm MUST be deterministic and it MUST output the same data
+compression algorithm and it MUST output the same data
```
Comment (vasilvv) [inline draft-ietf-tls-certificate-compression.md]: remove "and"

**Review by ghedo — APPROVED** [review]
Thanks, looks good to me, just added a comment to fix some wording.

**Review by vasilvv — COMMENTED** [review]
Thank you for doing this!

File: draft-ietf-tls-certificate-compression.md [hunk 5]
```diff
@@ -127,8 +128,9 @@ uncompressed_length
   enforce limits on the message size before performing decompression.
 
 compressed_certificate_message
-: The compressed body of the Certificate message, in the same format as it
-  would normally be expressed in. The compression algorithm defines how the
+: The result of applying the indicated compression algorithm to the encoded
+  Certificate message that would have be sent if certificate compression were not
```
Comment (ghedo) [inline draft-ietf-tls-certificate-compression.md]:
```suggestion
  Certificate message that would have been sent if certificate compression was not
```

**Review by ghedo — APPROVED** [review]
Almost, but I think I can accept my own suggestions so I'll just do that and merge.

**Pull request merged**